pulp_ansible/app/models.py

```diff
@@ -45,6 +46,13 @@ def relative_path(self):
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
         unique_together = ("version", "name", "namespace")
+        permissions = [
```

Comment:
Can we use the default django permissions for some of these? Django comes with create, update, view and delete by default

Reply:
Yes, they should already be in the db since `django.contrib.auth` is in `INSTALLED_APPS`.  I'll remove the extraneous code.